tests/helpers/periodPickerHelper-test.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * periodPickerHelper-test.js
+ * Created by Kwadwo Opoku-Debrah 12/07/18
+ */
+
+import * as periodPickerHelper from 'helpers/periodPickerHelper';
+
+const nativeDate = Date.now;
+
+const mockDate = (date) => {
+    // mock the current date
+    const month = parseInt(date.substring(5, 7), 10) - 1; // month is zero-indexed
+    const day = parseInt(date.substring(8));
+    const mock = new Date(date.substring(0, 4), month, day);
+    Date.now = () => mock;
+};
+
+afterAll(() => {
+    // restore the original, native date function
+    Date.now = nativeDate;
+});
+
+describe('periodPickerHelper', () => {
+    describe('handlePotentialStrings', () => {
+        it('should convert its input to an integer if it is a string', () => {
+            const output = periodPickerHelper.handlePotentialStrings('12');
+            expect(output).toEqual(12);
+        });
+        it('should return its input otherwise', () => {
+            const output = periodPickerHelper.handlePotentialStrings(12);
+            expect(output).toEqual(12);
+        });
+    });
+
+    describe('mostRecentPeriod', () => {
+        it('should return the most recently completed period', () => {
+            mockDate('1912-06-01');
+            const output = periodPickerHelper.mostRecentPeriod();
+            expect(output.period).toEqual(6);
+        });
+        it('should return the fiscal year of the most recently completed period', () => {
+            mockDate('1912-06-01');
+            const output = periodPickerHelper.mostRecentPeriod();
+            expect(output.year).toEqual(1912);
+        });
+        it('if GTAS data is not available, it should return the period before that', () => {
+            mockDate('1912-04-01');
```

Comment:
This test and the one before it should check dates within the same month, one before GTAS data becomes available (Jun 1 should return 7) and one after the data becomes available (on Jun 19 it should return 8)